platform/security/idp/security-idp-client/pom.xml

```diff
@@ -111,6 +126,7 @@
                         <Embed-Dependency>
                             common-system,
                             ddf-security-common,
+                            platform-security-session,
```

Comment:
Why is this getting embedded? This is a fragment bundle that attaches to Pax Web. This should not be here.